react/src/views/BoxEdit/components/BoxEdit.tsx

```diff
@@ -127,6 +129,16 @@ function BoxEdit({
     tags: boxData?.tags || [],
   };
 
+  if (boxData?.product?.deletedOn !== null) {
+    defaultValues.productId = { label: "", value: "" };
+    if (!toastIsActive!()) {
+      createToast({
+        message: "The product assigned to this box no longer exists. Please select another one.",
+        type: "warning",
+      });
+    }
```

Comment:
Requesting two changes: 
1. I would actually use a `FormErrorMessage` here rather than a toast, if possible. https://v1.chakra-ui.com/docs/components/form/form-control
2. Change the form error message to: "$deletedproductname has been deleted as a product type. Please select a different product."

NB: If it's not possible to do a `FormErrorMessage`, I'd like to change the toast type from `warning `to `info`. We try to avoid using the warning type for UX reasons.

Reply:
@serchtul To make it a FormErrorMessage you will either have to adjust the zod validation in line 65 or you create a separate error state just for this and pass it as an error besides the errors from react-hook-form to the SelectField in line 228.

I can imagine that it might be a bit of a tough nut and do not waste to much on it. If we do not find a solution, stay with the toast.